### PR TITLE
Remove: Couchbase.ServerEnterprise version 6.5.1

### PR DESCRIPTION
**Pull request by OfficialEsco**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/46466)

### CODE REVIEW

File: manifests/c/Couchbase/ServerEnterprise/6.5.1/Couchbase.ServerEnterprise.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created with YamlCreate.ps1 v2.0.7 $debug=AUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Couchbase.ServerEnterprise
+PackageVersion: 6.5.1
```
Comment (ImJoakim) [inline manifests/c/Couchbase/ServerEnterprise/6.5.1/Couchbase.ServerEnterprise.installer.yaml:5]:
```suggestion
PackageVersion: 6.5.6299
```

File: manifests/c/Couchbase/ServerEnterprise/6.5.1/Couchbase.ServerEnterprise.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.0.7 $debug=AUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Couchbase.ServerEnterprise
+PackageVersion: 6.5.1
```
Comment (ImJoakim) [inline manifests/c/Couchbase/ServerEnterprise/6.5.1/Couchbase.ServerEnterprise.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 6.5.6299
```

File: manifests/c/Couchbase/ServerEnterprise/6.5.1/Couchbase.ServerEnterprise.yaml [hunk 3]
```diff
@@ -1,21 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.7 $debug=AUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
 PackageIdentifier: Couchbase.ServerEnterprise
-Author: Couchbase, Inc.
-Publisher: Couchbase, Inc.
-PackageName: Couchbase Server Enterprise
 PackageVersion: 6.5.1
```
Comment (ImJoakim) [inline manifests/c/Couchbase/ServerEnterprise/6.5.1/Couchbase.ServerEnterprise.yaml:5]:
```suggestion
PackageVersion: 6.5.6299
```

File: manifests/c/Couchbase/ServerEnterprise/6.5.1/Couchbase.ServerEnterprise.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+PackageLocale: en-US
+Publisher: Couchbase, Inc.
```
Comment (ImJoakim) [inline manifests/c/Couchbase/ServerEnterprise/6.5.1/Couchbase.ServerEnterprise.locale.en-US.yaml:7]:
```suggestion
Publisher: Couchbase Inc.
```

File: manifests/c/Couchbase/ServerEnterprise/6.5.1/Couchbase.ServerEnterprise.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+Author: Couchbase, Inc.
+PackageName: Couchbase Server Enterprise
```
Comment (ImJoakim) [inline manifests/c/Couchbase/ServerEnterprise/6.5.1/Couchbase.ServerEnterprise.locale.en-US.yaml:12]:
```suggestion
PackageName: Couchbase Server Enterprise Edition
```

**Review by ImJoakim — CHANGES_REQUESTED** [review]
image

**Comment by ImJoakim** [comment]
Should probably add a `ShortDescription` too.

**Review by ItzLevvie — DISMISSED** [review]

**Comment by OfficialEsco** [comment]
last chance

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**